vscode-dotnet-runtime-library/src/Acquisition/SdkInstallationDirectoryProvider.ts

```diff
@@ -0,0 +1,19 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as fs from 'fs';
+import { IExtensionState } from '../IExtensionState';
+import { IInstallationDirectoryProvider } from './IInstallationDirectoryProvider';
+
+export class SdkInstallationDirectoryProvider implements IInstallationDirectoryProvider {
+    public getDotnetInstallDir(version: string, installDir: string): string {
+        return installDir;
+    }
+
+    public isBundleInstalled(dotnetPath: string, version: string, extensionState: IExtensionState, installingVersionsKey: string): boolean {
+        const installingVersions = extensionState.get<string[]>(installingVersionsKey, []);
+        return installingVersions.includes(version) && fs.existsSync(dotnetPath);
```

Comment:
I still feel the isBundleInstalled logic should probably not exist on this API but leaving a comment here about this logic. Why do you care about the `installingVersions` member here for SDKs and not for runtimes?

Reply:
Good call, I've updated to remove it. It doesn't make a difference in terms of functionality for runtimes because runtimes are all installed to different folders so the fs.exists check is enough. Since SDKs are all installed to the same folder, we have to do more when checking if a version is installed.

Reply:
I see, so is `installingVersions` effectively `installedVersions`? Maybe I was thrown off by the naming too

Reply:
> I see, so is installingVersions effectively installedVersions

Yes!

Reply:
Should change that at some point as I'm sure a future onlooker will be thoroughly confused like me 😄